### ISSUE TITLE
Apache Maven doesn't start up under Jato

### ISSUE THREAD

**Issue by penberg** [jatovm/jato#8]
```
~/testing/jato/apache-maven-3.0.3$ JAVACMD=/home/penberg/bin/jato ./bin/mvn
vm_class_resolve_class: warning: failed to load class org/apache/maven/eventspy/internal/EventSpyDispatcher
__convert_ldc: warning: unable to lookup class
convert_instruction: warning: conversion error at PC=157
[ERROR] Error executing Maven.
[ERROR] org.apache.maven.eventspy.internal.EventSpyDispatcher
```

**Comment by penberg** [comment]
This might be an issue with our 'ldc' JIT. It fails during compilation time if it's unable to load the class. However, it might be that we should fail during execution time as with fields. So we need to check the JVM spec and if my assuption is correct, use something similar as the static field fixups that use a guard page to lazily load classes.

**Comment by penberg** [comment]
Fixed by commit 8736a33.
